webv2/helpers/helpers.go

```diff
@@ -79,6 +79,24 @@ func createDatabase(ctx context.Context, uri string) error {
 				SchemaConversionObject JSON NOT NULL,
 				CreateTimestamp TIMESTAMP NOT NULL,
 			  ) PRIMARY KEY(VersionId)`,
+			  `CREATE TABLE JobExecutionData (
+				MigrationJobId STRING(50) NOT NULL,
+				SpannerDatabaseName STRING(50) NOT NULL,
+				AggMonitoringResources JSON,
```

Comment:
What all is stored in this field ? Please add a comment

Reply:
Not possible to put a comment here inline since it is a string. Added a comment at the top of the method.